lib/commands/config/generate.ts

```diff
@@ -46,6 +47,7 @@ export default class ConfigGenerateCmd extends Command {
 		Generate a config.json file for a device or fleet.
 
 		The target balenaOS version must be specified with the --version option.
+		The --dev option is used to select a development variant of balenaOS.
```

Comment:
I was more thinking something like (my wording definitely needs rework):
```suggestion
		The --dev option is used for generating a development mode configuration
```